easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.4.3-intel-2015a-ParMETIS-4.0.3.eb

```diff
@@ -0,0 +1,20 @@
+name = 'SuiteSparse'
+version = '4.4.3'
+
+homepage = 'http://www.cise.ufl.edu/research/sparse/SuiteSparse/'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True, 'static': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+maxparallel = 1
```

Comment:
in an ideal world, this would go below the `dependencies` line (with an empty line in between)

the only reason is style/consistency, but since we don't have an automated check for this yet, I'm not going to block this PR on this